packages/alfa-style/src/property/combinator/any.ts

```diff
@@ -0,0 +1,44 @@
+import { Parser } from "@siteimprove/alfa-parser";
+import { Err, Result } from "@siteimprove/alfa-result";
+
+export function any<I, T, E, A extends [] = []>(
```

Comment:
Should that be part of `alfa-parser` instead? It feels like it should be together with the other parser combinators (`either`, …)

Maybe need to break `alfa-parser/src/parser.ts` into specific files for individual combinators (or related ones, e.g. `pair`, `left`, `right` make sense together).

Reply:
It's a very CSS-specific combinator and so shouldn't live the generic combinators.